test/filesystems/blk_filesystem_test.rb

```diff
@@ -88,4 +88,60 @@
       end
     end
   end
+
+  describe "#in_network?" do
+    let(:disk) { Y2Storage::BlkDevice.find_by_name(fake_devicegraph, "/dev/sda") }
+
+    context "for a single disk in network" do
```

Comment:
In my opinion, this new version of the test is too coupled to the implementation (basically a 1:1 repetition of the code in the method). But let's not block the PR because of that.